### PR TITLE
[Mondrian] Tidy registration

### PR DESCRIPTION
**Pull request by dayo09**
This commit tidies registration of MondrianEditorProvider.

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

----

- For #1167

### CODE REVIEW

File: src/Mondrian/MondrianEditor.ts [hunk 1]
```diff
@@ -20,11 +20,15 @@ import {getUri} from '../Utils/external/Uri';
 
 /* istanbul ignore next */
 export class MondrianEditorProvider implements vscode.CustomTextEditorProvider {
-  public static register(context: vscode.ExtensionContext): vscode.Disposable {
```
Comment (dayo09) [inline src/Mondrian/MondrianEditor.ts:23]: From [#1167](https://github.com/Samsung/ONE-vscode/issues/1167#issue-1329445337),

The function is tidy as is, but I am trying to standardize the registration function as below. 😄 

> Point 1. introduce `public static register(context: vscode.ExtensionContext) : void` function
> Point 2. Register each modules with the static function call
> ```
> OneTreeDataProvider.register(context);
> ```

File: src/extension.ts [hunk 2]
```diff
@@ -69,7 +69,7 @@ export function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(disposableOneJsontracer);
 
-  context.subscriptions.push(MondrianEditorProvider.register(context));
+  MondrianEditorProvider.register(context);
```
Comment (dayo09) [inline src/extension.ts:72]: From clean code's view, a little bit of improvement is that it does registration at one place. (not passing the provider disposable to the caller)

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by llFreetimell** [comment]
@ai-moiseev 

As our release deadline is coming, and as this PR is quite simple, I would merge this PR without your approval :)
If you have any further opinion, please feel free to leave it!

**Pull request merged**